07-studio/01-quickstart.md

```diff
@@ -0,0 +1,295 @@
+toc: false
+---
+
+## Introduction
+
+This guide is here to take you from no knowledge of Studio or TypeDB to a place where you can freely run queries against some pre-provided data from entirely within Studio.
+By following each step, you will have the full picture on how to go from zero to full data visualisation using Studio and TypeDB.
+
+## What You Need
+- A [TypeDB Server](/docs/running-typedb/install-and-run) instance running locally on the default port (1729).
+- An OS-appropriate version of [Studio](https://vaticle.com/download#typedb-studio).
+
+## Running TypeDB Server
+Downloading, installing and running a TypeDB server differs slightly platform to platform, so we've created a guide to get you started here: [Install and Run TypeDB Server](/docs/running-typedb/install-and-run).
+
+## Connecting to TypeDB Server
+Once you launch studio, head to the top right-hand corner and click 'Connect to TypeDB'.
+
+<div class="slideshow">
+
+[slide:start]
+[header:start][header:end]
+[body:start]![Connection Manager Disconnected](/docs/images/studio/connection-interface-disconnected.png)[body:end]
+[footer:start]This is the connection manager. Fill in the address of your TypeDB Server. If you've followed the above guide to install and run TypeDB, this is probably `localhost:1729`.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![Connection Manager Connected](/docs/images/studio/connection-interface-connected.png)[body:end]
+[footer:start]Once you've connected successfully, the connection manager will reflect this in with the status field in the bottom left-hand corner. Additionally, the address will be reflected in the top right-hand corner of Studio.[footer:end]
+[slide:end]
+</div>
+
+Now Studio is connected to your TypeDB instance.
+
+## Creating a Database
+Now Studio and TypeDB Server are connected, but you haven't set up any databases. To do so, go to the databases icon in the top left-hand corner.
+
+<div class="slideshow">
+
+[slide:start]
+[header:start][header:end]
+[body:start]![Database Manager Empty](/docs/images/studio/studio-database.png)[body:end]
+[footer:start]This icon opens the database manager.[footer:end]
+[slide:end]
+
+[header:start][header:end]
+[body:start]![Database Manager Empty](/docs/images/studio/databases-interface-no-databases.png)[body:end]
+[footer:start]This is the database manager. Enter your database name then hit create. For the purposes of this example, we will use the name `github`.[footer:end]
+[slide:end]
+
+[slide:start]
+[body:start]![Database Manager With GitHub Database](/docs/images/studio/databases-interface-github-database.png)[body:end]
+[footer:start]Now you've successfully created a database named `github`![footer:end]
+[slide:end]
+</div>
+
+Select the database you just created by clicking the dropdown menu titled 'Select Databases' immediately right of the databases icon.
+
+[//]: # (<img height='1em' width='1em' src="/docs/images/studio/databases-icon.png"></img>.)
```

Comment:
Is this something we've done elsewhere in the docs or is it new?

There are some bugs that occur only in production (and not when doing `jekyll serve` or even `jekyll build` locally), so it's always worth doing a close review of the page once it's in production especially if new concepts are introduced. But this looks fine and may be useful in the future if it works.

Reply:
This was a comment generated by Intellij CMD+/. Should be doing nothing but have removed now.